charactersheet/charactersheet/index.html

```diff
@@ -461,30 +461,6 @@ <h4 class="modal-title" id="exportModalLabel" data-dismiss="modal">Export</h4>
   </div>
 </div>
 
-<!-- Import Modal Form -->
-<div class="modal fade" id="importModal" tabindex="-1" role="dialog"
-  aria-labelledby="importModalLabel" data-bind="with: wizardViewModel">
-  <div class="modal-dialog" role="document">
-    <div class="modal-content container-fluid">
-      <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h4 class="modal-title" id="importModalLabel">Import</h4>
-      </div>
-      <div class="modal-body">
-        <p>If you've exported you character data before, you can import it here!</p>
-        <!-- Add import form. -->
-        <input type="file" id="file" name="file" data-bind="file: { data: fileContents, reader: fileReader }"/>
-				<br>
-				<p>Or import it from your Dropbox!</p>
-					<div id="dropbox-container"></div>
-      </div>
-      <div class="modal-footer">
-        <button type="button" class="btn btn-primary" data-bind="click: importFromFile" data-dismiss="modal">Import</button>
-      </div>
-    </div>
-  </div>
-</div>
-
 <script type="text/javascript" src="https://www.dropbox.com/static/api/2/dropins.js" id="dropboxjs" data-app-key="7bvlboabwiprcys"></script>
```

Comment:
I created a key using the AC Dropbox resource account. Here it is:
```
<script type="text/javascript" src="https://www.dropbox.com/static/api/2/dropins.js" id="dropboxjs" data-app-key="3j1z94debj5xcwn"></script>
```

Could you please add this new key?